bin/node-template-archive/src/cli_opts.rs

```diff
@@ -14,53 +14,52 @@
 // You should have received a copy of the GNU General Public License
 // along with substrate-archive.  If not, see <http://www.gnu.org/licenses/>.
 
-use clap::{load_yaml, App};
 use std::path::PathBuf;
 
+use clap::{load_yaml, App};
+
 #[derive(Clone)]
 pub struct CliOpts {
-    pub file: PathBuf,
-    pub log_level: log::LevelFilter,
-    pub chain_spec: node_template::chain_spec::ChainSpec,
+	pub file: PathBuf,
```

Comment:
please use the [.editconfig](https://github.com/paritytech/substrate-archive/blob/master/.editorconfig) for `rustfmt`, we use `tabs` instead of spaces for example.

Reply:
Yeah, I know. Actually, the original indent is space, and the modified one is tab

Reply:
Ok, sorry my bad. These may have been missed because they are excluded for the workspace.